magenta-lib/src/main/scala/magenta/deployment_type/CloudFormation.scala

```diff
@@ -172,6 +172,9 @@ class CloudFormation(vcsUrlLookup: VcsLookup) extends DeploymentType with CloudF
     // The tag name here ('gu:repo') and format ('guardian/:reponame') MUST
     // mirror the tag name and format used by @guardian/cdk.
     val guRepoTag = ("gu:repo" -> vcsUrlLookup.get(target.parameters.build.projectName, target.parameters.build.id))
+    if (guRepoTag._2 == "") {
```

Comment:
Forgot about the `._` syntax in Scala 🙈.

Reply:
Yeah, it's horrible :(

Reply:
I'll lift it into a variable to make it more readable.